docs/source/concepts/index.rst

```diff
@@ -7,16 +7,7 @@ Here we cover several core conceptual topics related to modeling with
 the Vivarium framework.
 
 .. toctree::
+   :glob:
```

Comment:
Glob should keep this section up to date so we don't forget to add paths to the new concept docs.